src/CreeDictionary/API/search/presentation.py

```diff
@@ -71,9 +89,19 @@ class PresentationResult:
     presentation things like labels.
     """
 
-    def __init__(self, result: types.Result, *, search_run: core.SearchRun):
+    def __init__(
+        self,
+        result: types.Result,
+        *,
+        search_run: core.SearchRun,
+        display_mode="community",
```

Comment:
Note: I had to punch holes through a lot of systems to get the current display mode from the `HttpRequest` down here where it is used to actually do the relabelling. The relabelling cannot (easily) be done in the template -- I tried 😭 😭 😭